content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+
+lm_dv_cases_df = dv_cases_df %>%
+  group_by(geo_value) %>%
+  summarize(b=tryCatch(lm(cases ~ dv)[["coefficients"]][[1]],error=function(x){NA}),
+            m=tryCatch(lm(cases ~ dv)[["coefficients"]][[2]],error=function(x){NA})) %>%
+  ungroup()
+```
+
+```{r, message=FALSE,include=FALSE}
+rank_cor = function(x, y) {
+  return(cor(x, y, method="spearman",use="na.or.complete"))
+}
+
+ggplot(inner_join(dv_cases_df,lm_dv_cases_df, by="geo_value") %>%
+         group_by(time_value) %>%
+         summarize(corr_out = rank_cor(dv*slope_6wk + int_6wk, cases),
+                   corr_in = rank_cor(dv*m + b, cases),
+                   corr_orig = rank_cor(dv,cases), .groups="drop")) +
+  geom_line(aes(time_value,corr_in,color="Single Correction (in sample)")) +
+  geom_line(aes(time_value,corr_out,color="\"Real Time\" Correction")) +
+  geom_line(aes(time_value,corr_orig,color="Original Signal")) +
+  labs(x="Date",y="Correlation",title="Space-wise Correlation of DV-Corrected and Case Incidence",subtitle="Over all counties with at least 500 cumulative cases") +
+  theme(legend.position = "bottom", legend.title = element_blank())
+```
+
+From the graphs above, we see that correcting the DV signal by turning it into a sensor greatly improved the space-wise correlations! Furthermore, the correlation is relatively constant over time, demonstrating that the data quality did not necessarily decrease over time, since a simple linear correction stopped the deterioration.
+
+### Open Questions and Future Directions
+We were happy to see that a location-specific correction was able to greatly increase the space-wise correlation. By looking at the correlations of the two corrections in the graph above, we can see that they perform quite differently early on, and perform more similarly after mid-July. This indicates that the regression coefficients of the real-time correction, which are fit using a sliding window, change significantly over time. We would like to look into this non-stationary relationship between the DV signal and case incidence to better understand why it changes, and why counties in New York and New Jersey have such a different relationship in the first place.
+
+We would also like to see how this data compares to that of other healthcare insurance providers. Some insurance providers cover different populations with different rates, which could make their signal more or less correlated with the reported case incidence. We would expect that by combining datasets and covering more people, we can produce a more useful signal.
+
+We believe that this dataset will be useful for forecasting Covid cases and deaths, since a doctor visit usually precedes a confirmed test and death. We can compare time-wise and space-wise correlations for cases lagged by a week or two, as well as deaths. Forecasting models can also take the current DV signal as a feature to help predict what the case incidence will be several weeks into the future.
```

Comment:
Comparing lagged correlations is simple enough that we should do it and add it to the post.